### DISCUSSION TITLE
Press Key does not work for browser hotkeys

### DISCUSSION THREAD

**Discussion by emp-00** [AutomaApp/automa#1758] [Q&A]
I would like to use Automa to activate a different extension by pressing the defined hotkey. This does not work - tested with 0, 50, 100 and 500ms press time and various key combinations such as Shift-Ctrl-S or Ctrl-P. Even simply F1 (opening the help page) does not work - no error in the log, just claiming "success". However, nothing happens just as if the hotkey is NOT pressed. What am I doing wrong? Please help.

![image](https://github.com/AutomaApp/automa/assets/60714349/860d0f4e-6715-41f7-8fb0-3eb416f2caf7)
![image](https://github.com/AutomaApp/automa/assets/60714349/78678412-4e37-4d82-af79-dd6a755bc649)

**Comment by amaillet2024** [comment]
Hello same problem as you
I need to submit a form but nothing happen, so I tried to simulate ctrl+enter but nothing happen
either through press key or trigger envent et click element. 
Did you solve your problem ?
